packages-best-practices.md

```diff
@@ -0,0 +1,40 @@
+---
+description: ' Best practices for how and when to create a Python package. '
+title: Python Packages Best Practices
+---
+
+This page contains steps on when and how to create a Python package.
+
+## When Can a Package be Made?
+
+A Python package can be created when it meets the following conditions:
+
+1. It contains multiple reusable modules.
+1. Multiple tests running in the Open 3D Engine pipeline depend on the package.
+1. The tests or modules cannot be moved to relatively import the package.
+
+Consider adding a module to an already existing package instead of creating a new one.
+
+## How to Make a Package
```

Comment:
Use sentence casing for titles for consistency (since the first one is sentence-case, now).

```suggestion
## How to make a Python package
```

Reply:
Fixed title casing